web/packages/teleport/src/Discover/Kubernetes/EnrollEKSCluster/EnrollEksCluster.tsx

```diff
@@ -295,54 +389,95 @@ export function EnrollEksCluster(props: AgentStepProps) {
         clear={clear}
         disableSelector={fetchClustersAttempt.status === 'processing'}
       />
-      {!hasIamPermError && tableData.currRegion && (
+      {showContent && (
         <>
           <Box mb={2}>
-            <Toggle
-              isToggled={isAppDiscoveryEnabled}
-              onToggle={() => setAppDiscoveryEnabled(!isAppDiscoveryEnabled)}
-            >
-              <Box ml={2} mr={1}>
-                Enable Kubernetes App Discovery
-              </Box>
-              <ToolTipInfo>
-                Teleport's Kubernetes App Discovery will automatically identify
-                and enroll to Teleport HTTP applications running inside a
-                Kubernetes cluster.
-              </ToolTipInfo>
-            </Toggle>
+            <Flex alignItems="center" gap={3}>
+              <Toggle
+                isToggled={isAutoDiscoveryEnabled}
+                onToggle={() =>
+                  setAutoDiscoveryEnabled(!isAutoDiscoveryEnabled)
+                }
+              >
+                <Box ml={2} mr={1}>
+                  Auto-enroll all EKS clusters for selected region.
```

Comment:
we dont period the other one 😅 
```suggestion
                  Auto-enroll all EKS clusters for selected region
```